packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -2482,6 +2482,25 @@ test('addVaultType: extra, unexpected params', async t => {
   t.true(matches(actual, M.remotable()), 'unexpected params are ignored');
 });
 
+test.todo('storage keys with storageNode/marshaller');
```

Comment:
@michaelfig are there any tests like this yet? I found `makeFakeStorage` but nothing for marshaller. I guess I don't need that to test the keys but I'm trying to have tests for this stored sub feature overall